src/main/java/org/sasanlabs/service/vulnerability/sampleVulnerability/SampleVulnerability.java

```diff
@@ -0,0 +1,108 @@
+package org.sasanlabs.service.vulnerability.sampleVulnerability;
```

Comment:
Please don't checkin sample vulnerability related files. Check in only the test file.